index.html

```diff
@@ -0,0 +1,1796 @@
+                        <div class="row no-gutters magazine">
+                            <div class="col-lg-4 col-md-4 col-sm-12">
+                                <img src="https://d.newsweek.com/en/full/1585087/may-08-15-2020-cover.jpg?w=170&amp;q=95&amp;f=c4d10af6474d34e586607b280b6589f0" class="card-img" alt="...">
+                            </div>
+                            <div class="col-lg-8 col-md-8 col-sm-12">
+                                <div class="card-body">
+                                    <h5 class="card-title">In-depth Stories You Don't See Elsewhere About Topics You Want to Read!</h5>
+                                    <p class="card-text text-muted">All Access + Weekly Delivery.Choose A Membership That's Perfect for You!</p>
+                                    <a href="#" class="btn btn-light text-danger">SUBSCRIBE&nbsp;&nbsp;></a>
+                                </div>
+                            </div>  
+                        </div>
+                    </article>
+                    <article class="col-lg-5 col-md-12 col-sm-12">
+                        <div class="text-danger font-weight-bold">NEWSLETTER</div>
+                        <div class="row no-gutters">
+                            <div class="col-lg-4 col-md-4 col-sm-12">
+                                <img src="https://d.newsweek.com/en/full/1541302/starting5-homepage.png?w=397&amp;h=265&amp;q=90&amp;f=742bce9a6f72dd8f19ba2e7fd5db06e5" style="height: 200px;" class="card-img" alt="...">
+                            </div>
+                            <div class="col-lg-8 col-md-8 col-sm-12">
+                                <div class="card-body">
+                                    <h5 class="card-title">The Starting 5</h5>
+                                    <p class="card-text text-muted">See why nearly a quarter of a million subscribers begin their day with the Starting 5.</p>
+                                    <a href="#" class="btn btn-danger">SIGN UP&nbsp;&nbsp;></a>
+                                </div>
+                            </div>  
+                        </div>
+                    </article>
+                </div>
+                <footer class="footer">
+                    <div class="d-flex footer1">
+                        <div class="flex-fill">
+                            <span id="logo-footer">Newsweek</span>
+                            <span id="Newsweek-2020">© 2020 NEWSWEEK</span>
+                        </div>
+                        <div class="align-self-center footer-icons">
+                            <i class="fa fa-facebook-square" aria-hidden="true"></i>
+                            <i class="fa fa-twitter" aria-hidden="true"></i>
+                            <i class="fa fa-instagram" aria-hidden="true"></i>
+                            <i class="fa fa-tumblr" aria-hidden="true"></i>
+                            <i class="fa fa-linkedin" aria-hidden="true"></i>
```

Comment:
Each of these icons should be enclosed by links (```<a href="#"><i ... </i></a>```). In the original webpage when you hover them, they change the colors. It's a nice detail that you can copy!

Also, in phone version, they are below the Newsweek logo.

> Original
![image](https://user-images.githubusercontent.com/57414789/81482699-9d008880-91fe-11ea-9d06-de381f6604c1.png)

> Your page
![image](https://user-images.githubusercontent.com/57414789/81482738-dcc77000-91fe-11ea-8abb-dd30ac98cc52.png)